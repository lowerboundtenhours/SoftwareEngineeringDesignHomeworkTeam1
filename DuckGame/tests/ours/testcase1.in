duck MallardDuck CanFly Quack
swim MallardDuck
fly MallardDuck
quack MallardDuck
changeFly MallardDuck CannotFly
fly MallardDuck
changeQuack MallardDuck Squeak
quack MallardDuck
changeQuack MallardDuck Silent
quack MallardDuck