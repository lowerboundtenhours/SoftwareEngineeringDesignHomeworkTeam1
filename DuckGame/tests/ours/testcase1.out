MallardDuck is swimming
duck is flying
quack quack quack
duck cannot fly
squeak squeak squeak
(silent)